patches.json

```diff
@@ -241,6 +241,9 @@
     "Sitemap Generation Warnings": {
       ">=2.3.0 <2.3.2": "MCLOUD-3025__sitemap_generation_warnings__2.3.0.patch",
       ">=2.3.2 <2.3.5": "MCLOUD-3025__sitemap_generation_warnings__2.3.3.patch"
+    },
+    "Fix load balancer issue": {
+      ">=2.3.4 <2.3.6": "MCLOUD-5837__fix_filesystem_load_balancer_issue.patch"
```

Comment:
To follow patch naming standard, can you please add number from what version this patch should be applied? It should follow after double underscore at he end of file name - `MCLOUD-5837__fix_filesystem_load_balancer_issue__2.3.4.patch`

Reply:
Done